internal/kubernetes/operator/project/project_test.go

```diff
@@ -98,14 +99,14 @@ func TestBuildAtlasProject(t *testing.T) {
 			},
 		}
 
-		encryptionAtRest := &mongodbatlas.EncryptionAtRest{
-			GroupID:       "TestGroupID",
-			AwsKms:        mongodbatlas.AwsKms{},
-			AzureKeyVault: mongodbatlas.AzureKeyVault{},
-			GoogleCloudKms: mongodbatlas.GoogleCloudKms{
+		encryptionAtRest := &atlasv2.EncryptionAtRest{
+			//GroupID:       "TestGroupID",
```

Comment:
We can safely clean-up this 
![Delete](https://thumbs.gfycat.com/AlertLegitimateAlligator-size_restricted.gif)